packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/AdminContestsPage/AdminContestsPage.tsx

```diff
@@ -55,7 +72,10 @@ const AdminContestsPage = () => {
       event: MixpanelContestEvents.CREATE_CONTEST_BUTTON_PRESSED,
       isPWA: isAddedToHomeScreen,
     });
-    navigate('/manage/contests/launch');
+
+    return farcasterContestEnabled
+      ? setContestView(ContestView.TypeSelection)
+      : navigate('/manage/contests/launch');
```

Comment:
i think we should also update the mixpanel logic here, i.e only send mixpanel event when launch is clicked for the content type selected below